shared/constants.ts

```diff
@@ -1,2 +1,3 @@
-export const minDenoVersion = '1.10.1'
-export const defaultReactVersion = '17.0.2'
+export const minDenoVersion = "1.10.1";
+export const defaultReactVersion = "17.0.2";
+export const supportedAlephTemplate = ["hello-world"];
```

Comment:
we can check the downloaded zip that includes all the templates via dir name

Reply:
@ije I can check template-status from https://github.com/alephjs/alephjs-templates, i will be very fast ?